### PR TITLE
Remove old, add new credentials endpoints

### PR DESCRIPTION
**Pull request by ricardosantacattarina**
Remove old endpoints for credentials creation
Add new endpoints for credentials creation, read, read events and delete.

### CODE REVIEW

**Comment by ricardosantacattarina** [comment]
[[User Management] Add new endpoints from CFV and JD](https://trello.com/c/RBNCj7U8/1175-user-management-add-new-endpoints-from-cfv-and-jd)

File: Leaf API.postman_collection.json [hunk 1]
```diff
@@ -1421,6 +1026,631 @@
...
 							]
 						},
+						{
+							"name": "Provider Credentials",
+							"item": [
+								{
+									"name": "ClimateFieldView",
+									"item": [
+										{
+											"name": "Get credential of LeafUser",
```
Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:1036]: Lest include the provider name in the title of this endpoint

File: Leaf API.postman_collection.json [hunk 2]
```diff
@@ -1421,6 +1026,631 @@
...
+														"users",
+														"{{leaf_user_id}}",
+														"climate-field-view-credentials"
+													]
+												}
+											},
+											"response": []
+										},
+										{
+											"name": "Add credential to LeafUser",
```
Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:1083]: Lest include the provider name in the title of this endpoint

File: Leaf API.postman_collection.json [hunk 3]
```diff
@@ -1421,6 +1026,631 @@
...
+														"users",
+														"{{leaf_user_id}}",
+														"climate-field-view-credentials"
+													]
+												}
+											},
+											"response": []
+										},
+										{
+											"name": "Delete credential of LeafUser",
```
Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:1139]: Lest include the provider name in the title of this endpoint

File: Leaf API.postman_collection.json [hunk 4]
```diff
@@ -1421,6 +1026,631 @@
...
+											},
+											"response": []
+										}
+									]
+								},
+								{
+									"name": "JohnDeere",
+									"item": [
+										{
+											"name": "Get credential of LeafUser",
```
Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:1191]: Lest include the provider name in the title of this endpoint

File: Leaf API.postman_collection.json [hunk 5]
```diff
@@ -1421,6 +1026,631 @@
...
+														"users",
+														"{{leaf_user_id}}",
+														"john-deere-credentials"
+													]
+												}
+											},
+											"response": []
+										},
+										{
+											"name": "Add credential to LeafUser",
```
Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:1238]: Lest include the provider name in the title of this endpoint

File: Leaf API.postman_collection.json [hunk 6]
```diff
@@ -1421,6 +1026,631 @@
...
+														"users",
+														"{{leaf_user_id}}",
+														"john-deere-credentials"
+													]
+												}
+											},
+											"response": []
+										},
+										{
+											"name": "Delete credential of LeafUser",
```
Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:1294]: Lest include the provider name in the title of this endpoint

File: Leaf API.postman_collection.json [hunk 7]
```diff
@@ -1421,6 +1026,631 @@
...
+											},
+											"response": []
+										}
+									]
+								},
+								{
+									"name": "Trimble",
+									"item": [
+										{
+											"name": "Get credential of LeafUser",
```
Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:1346]: Lest include the provider name in the title of this endpoint

File: Leaf API.postman_collection.json [hunk 8]
```diff
@@ -1421,6 +1026,631 @@
...
+														"users",
+														"{{leaf_user_id}}",
+														"trimble-credentials"
+													]
+												}
+											},
+											"response": []
+										},
+										{
+											"name": "Add credential to LeafUser",
```
Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:1393]: Lest include the provider name in the title of this endpoint

File: Leaf API.postman_collection.json [hunk 9]
```diff
@@ -1421,6 +1026,631 @@
...
+														"users",
+														"{{leaf_user_id}}",
+														"trimble-credentials"
+													]
+												}
+											},
+											"response": []
+										},
+										{
+											"name": "Delete credential of LeafUser",
```
Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:1449]: Lest include the provider name in the title of this endpoint

File: Leaf API.postman_collection.json [hunk 10]
```diff
@@ -1421,6 +1026,631 @@
...
+											},
+											"response": []
+										}
+									]
+								},
+								{
+									"name": "CNHi",
+									"item": [
+										{
+											"name": "Get credential of LeafUser",
```
Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:1501]: Lest include the provider name in the title of this endpoint

File: Leaf API.postman_collection.json [hunk 11]
```diff
@@ -1421,6 +1026,631 @@
...
+														"users",
+														"{{leaf_user_id}}",
+														"cnhi-credentials"
+													]
+												}
+											},
+											"response": []
+										},
+										{
+											"name": "Add credential to LeafUser",
```
Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:1548]: Lest include the provider name in the title of this endpoint

File: Leaf API.postman_collection.json [hunk 12]
```diff
@@ -1421,6 +1026,631 @@
...
+														"users",
+														"{{leaf_user_id}}",
+														"cnhi-credentials"
+													]
+												}
+											},
+											"response": []
+										},
+										{
+											"name": "Delete credential of LeafUser",
```
Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:1604]: Lest include the provider name in the title of this endpoint

**Review by fabyan-alexander — CHANGES_REQUESTED** [review]
Nice work! I would like to include the provider name in the endpoints because we cannot see the group of a "open" endpoint in postman.

File: Leaf API.postman_collection.json (hunk 1 above)
Comment (ricardosantacattarina) [reply Leaf API.postman_collection.json]: Added it: https://github.com/Leaf-Agriculture/Leaf-API-Postman-Collection/pull/44/commits/8194d7cb4685389a79c930a74168de128fbfb605

File: Leaf API.postman_collection.json (hunk 2 above)
Comment (ricardosantacattarina) [reply Leaf API.postman_collection.json]: Added it: https://github.com/Leaf-Agriculture/Leaf-API-Postman-Collection/pull/44/commits/8194d7cb4685389a79c930a74168de128fbfb605

File: Leaf API.postman_collection.json (hunk 3 above)
Comment (ricardosantacattarina) [reply Leaf API.postman_collection.json]: Added it: https://github.com/Leaf-Agriculture/Leaf-API-Postman-Collection/pull/44/commits/8194d7cb4685389a79c930a74168de128fbfb605

File: Leaf API.postman_collection.json (hunk 4 above)
Comment (ricardosantacattarina) [reply Leaf API.postman_collection.json]: Added it: https://github.com/Leaf-Agriculture/Leaf-API-Postman-Collection/pull/44/commits/8194d7cb4685389a79c930a74168de128fbfb605

File: Leaf API.postman_collection.json (hunk 5 above)
Comment (ricardosantacattarina) [reply Leaf API.postman_collection.json]: Added it: https://github.com/Leaf-Agriculture/Leaf-API-Postman-Collection/pull/44/commits/8194d7cb4685389a79c930a74168de128fbfb605

File: Leaf API.postman_collection.json (hunk 6 above)
Comment (ricardosantacattarina) [reply Leaf API.postman_collection.json]: Added it: https://github.com/Leaf-Agriculture/Leaf-API-Postman-Collection/pull/44/commits/8194d7cb4685389a79c930a74168de128fbfb605

File: Leaf API.postman_collection.json (hunk 7 above)
Comment (ricardosantacattarina) [reply Leaf API.postman_collection.json]: Added it: https://github.com/Leaf-Agriculture/Leaf-API-Postman-Collection/pull/44/commits/8194d7cb4685389a79c930a74168de128fbfb605

File: Leaf API.postman_collection.json (hunk 8 above)
Comment (ricardosantacattarina) [reply Leaf API.postman_collection.json]: Added it: https://github.com/Leaf-Agriculture/Leaf-API-Postman-Collection/pull/44/commits/8194d7cb4685389a79c930a74168de128fbfb605

File: Leaf API.postman_collection.json (hunk 9 above)
Comment (ricardosantacattarina) [reply Leaf API.postman_collection.json]: Added it: https://github.com/Leaf-Agriculture/Leaf-API-Postman-Collection/pull/44/commits/8194d7cb4685389a79c930a74168de128fbfb605

File: Leaf API.postman_collection.json (hunk 10 above)
Comment (ricardosantacattarina) [reply Leaf API.postman_collection.json]: Added it: https://github.com/Leaf-Agriculture/Leaf-API-Postman-Collection/pull/44/commits/8194d7cb4685389a79c930a74168de128fbfb605

File: Leaf API.postman_collection.json (hunk 12 above)
Comment (ricardosantacattarina) [reply Leaf API.postman_collection.json]: https://github.com/Leaf-Agriculture/Leaf-API-Postman-Collection/pull/44/commits/8194d7cb4685389a79c930a74168de128fbfb605

File: Leaf API.postman_collection.json [hunk 13]
```diff
@@ -1421,6 +1026,823 @@
...
+														"usermanagement",
+														"api",
+														"users",
+														"{{leaf_user_id}}",
+														"climate-field-view-credentials",
+														"events"
+													]
+												}
+											},
+											"response": []
```
[the remark this answers is not in the document]
Comment (ricardosantacattarina) [reply Leaf API.postman_collection.json]: Removed it: https://github.com/Leaf-Agriculture/Leaf-API-Postman-Collection/pull/44/commits/cfa86915645476f76bfc0accbda3b6c209f0fd32

File: Leaf API.postman_collection.json [hunk 14]
```diff
@@ -1421,6 +1026,823 @@
...
+														"services",
+														"usermanagement",
+														"api",
+														"users",
+														"{{leaf_user_id}}",
+														"john-deere-credentials",
+														"events"
+													]
+												}
+											},
```
[the remark this answers is not in the document]
Comment (ricardosantacattarina) [reply Leaf API.postman_collection.json]: Removed it: https://github.com/Leaf-Agriculture/Leaf-API-Postman-Collection/pull/44/commits/cfa86915645476f76bfc0accbda3b6c209f0fd32

File: Leaf API.postman_collection.json [hunk 15]
```diff
@@ -1421,6 +1026,823 @@
...
+														"usermanagement",
+														"api",
+														"users",
+														"{{leaf_user_id}}",
+														"trimble-credentials",
+														"events"
+													]
+												}
+											},
+											"response": []
```
[the remark this answers is not in the document]
Comment (ricardosantacattarina) [reply Leaf API.postman_collection.json]: Removed it: https://github.com/Leaf-Agriculture/Leaf-API-Postman-Collection/pull/44/commits/cfa86915645476f76bfc0accbda3b6c209f0fd32

File: Leaf API.postman_collection.json [hunk 16]
```diff
@@ -1421,6 +1026,823 @@
...
+														"usermanagement",
+														"api",
+														"users",
+														"{{leaf_user_id}}",
+														"cnhi-credentials",
+														"events"
+													]
+												}
+											},
+											"response": []
```
[the remark this answers is not in the document]
Comment (ricardosantacattarina) [reply Leaf API.postman_collection.json]: Removed it: https://github.com/Leaf-Agriculture/Leaf-API-Postman-Collection/pull/44/commits/cfa86915645476f76bfc0accbda3b6c209f0fd32

**Review by fabyan-alexander — APPROVED** [review]
Nice job!!!! 👏 👏 👏

**Pull request merged**